src/Components/v2/ArtistBio.tsx

```diff
@@ -16,20 +15,15 @@ export const MAX_CHARS = {
 }
 
 export class ArtistBio extends React.Component<ArtistBioProps> {
-  renderReadMore = size => {
-    return (
-      <ReadMore
-        onReadMoreClicked={this.props.onReadMoreClicked}
-        maxChars={size === "xs" ? MAX_CHARS.xs : MAX_CHARS.default}
-        content={this.props.bio.biography_blurb.text}
-      />
-    )
-  }
   render() {
+    const { bio } = this.props
     return (
       <Serif size="3">
-        <Media at="xs">{this.renderReadMore("xs")}</Media>
-        <Media greaterThan="xs">{this.renderReadMore("rest")}</Media>
+        <span
+          dangerouslySetInnerHTML={{
+            __html: bio.biography_blurb.text,
+          }}
```

Comment:
Hmmm... do we have to do this? Does the bio have html contents?

Reply:
I think my concern is this opens us up to persisted XSS attacks if a partner can add bio text.